string SWITCH_TO_ACTION = switch-to-action
string SELECT_STEP = select-step
string DESELECT_STEP = deselect-step
string DELETE_STEP = delete-step
string SET_LOOP_STEP = set-loop-step
string SET_NO_LOOP_STEP = set-no-loop-step
string SET_STRATEGY = set-strategy
string SET_ARM_STEP_STRATEGY = set-arm-step-strategy
string SET_IGNORE_CONDITIONS = set-ignore-conditions
string SET_NO_IGNORE_CONDITIONS = set-no-ignore-conditions
string SET_OBJECT_SIMILARITY_THRESHOLD = set-object-similarity-threshold
string SET_CONDITION_ORDER = set-condition-order
string command
uint8 param
int32[] param_list
float32 param_float